package.json

```diff
@@ -21,6 +22,7 @@
   },
   "dependencies": {
     "@theforeman/vendor": "^4.14.0",
+    "eslint-plugin-spellcheck": "0.0.17",
```

Comment:
This needs adding to https://github.com/theforeman/foreman-packaging/blob/rpm/develop/update-requirements#L10, but otherwise LGMT from a packaging perspective.

Reply:
thanks, opened https://github.com/theforeman/foreman-packaging/pull/5845